File: wio_terminal/lib/song_info.cpp
#ifndef SONG_INFO_H
#define SONG_INFO_H


class SongInfo{
    public:

        SongInfo(String name, String fileName){
            this->name = name;
            this->fileName = fileName;
        }

        SongInfo(){
            
        }


        String getName(){
            return name;
        }

        String getFileName(){
            return fileName;
        }

    private:
        String name;
        String fileName;
};


#endif

File: android/app/src/main/java/com/example/home4u/music/SongInfo.java
package com.example.home4u.music;

public class SongInfo {
    private final String fileName;
    private final String name;
    private final String artist;
    private final String album;
    private final String length;


    SongInfo(String fileName, String name, String artist, String album, String length) {
        this.fileName = fileName;
        this.name = name;
        this.artist = artist;
        this.album = album;
        this.length = length;
    }


    public String getFileName() {
        return fileName;
    }

    public String getName() {
        return name;
    }
}
